src/tests/multihost/alltests/test_services.py

```diff
@@ -88,15 +88,30 @@ def test_0003_bz1713368(self, multihost):
         """
         # sssd-dbus is a weak dependency of sssd-tools package so we did not
         # get sssd-dbus in '# yum deplist sssd-tools' command
-        rpm_grep = 'yum repoquery --recommends sssd-tools'
-        cmd = multihost.client[0].run_command(rpm_grep, raiseonerr=False)
-        if cmd.returncode == 0:
-            status = 'PASS'
-            find = re.compile(r'sssd-dbus')
-            result = find.search(cmd.stdout_text)
-            if result is None:
-                status = 'FAIL'
-            assert status != 'FAIL'
+        command = "sed -rn 's/.*([0-9])\.[0-9].*/\\1/p' /etc/redhat-release"
+        version = [int(i) for i in
+                   multihost.client[0].run_command(command).stdout_text.split()
+                   if i.isdigit()][0]
```

Comment:
can we user self.multihost.distro to get the rhel version?
https://github.com/SSSD/sssd/blob/master/src/tests/multihost/sssd/testlib/common/utils.py#L72

Reply:
Fixed